lib/jetstream/producer/jetstream_acknowledger.ex

```diff
@@ -0,0 +1,14 @@
+with {:module, _} <- Code.ensure_compiled(Broadway) do
```

Comment:
This module ensures we "own" the functionality even though the implementation here is just to delegate. This is to follow the principle of not mocking something we don't own